src/linker/utilities/data_utils.py

```diff
@@ -51,3 +51,11 @@ def validate_dummy_output(filepath: Path) -> None:
         raise LookupError(
             f"Data file {filepath} is missing required column(s) {missing_columns}"
         )
+
+
+def validate_dummy_input(filepath: Path) -> Optional[List[str]]:
```

Comment:
I moved this back into data utils; eventually if there are lots of very different validation functions we should have a "validations" file or something